builds/ob-release-notes.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="ob-release-notes"]
+= Release notes
+include::_attributes/common-attributes.adoc[]
+:context: ob-release-notes
+
+toc::[]
+
+Release notes contain information about new and deprecated features, breaking changes, and known issues. The following release notes apply for the most recent {builds-shortname} releases on {ocp-product-title}.
```

Comment:
"breaking" sounds like marketing jargon. How about "latest"?

Reply:
breaking changes basically point to issues or things that might break your code. For example: https://docs.openshift.com/pipelines/1.11/about/op-release-notes.html#breaking-changes-1-11_op-release-notes